public/assets/css/style.css

```diff
@@ -726,4 +740,29 @@ button {
         max-width: 30vw;
         margin: auto;
     }
+    .home{
+        height: calc(100vh - 2.5rem);
+    }
+    .unlock-h1{
+        font-size: 4rem;
+    }
+    .unlock-h2{
+        font-size: 3rem;
+    }
+    .slide{
+        max-width: 450px;
+        max-height: 270px;
+    }
+    .slide-img{
+        max-width: 270px;
+    }
+    .rotate{
+        width: 270px;
+        font-size: 2vw;
+    }
+    .front{
+        transform: rotate(-130deg) translate(-10rem, 50rem);
+        height: 50vw;
+        padding-top: 50rem;
```

Comment:
too much padding here no more than 4 rem, if you wanna center, use flexbox or something else !